src/ids/index.ts

```diff
@@ -315,6 +315,21 @@ function parseGestaltId(data: any): GestaltId {
   return ['identity', [providerId, identityId]];
 }
 
+function parseGestaltIdentityId(data:any) : ProviderIdentityId {
```

Comment:
If it's returning a `ProviderIdentityId` then it should be called `parseProviderIdentityId`.

Reply:
@CMCDragonkai

Reply:
It is actually a `GestaltIdentityId`. That return type is incorrect.

Reply:
Sorry there's no type defined for `GestaltIdentityId`. It is actually just `GestaltId`.

Reply:
```
type GestaltId = ['node', NodeId] | ['identity', ProviderIdentityId];
```

I want to keep it as `parseGestaltIdentityId` since `provider:identity` is part of a gestalt ID, not `ProviderIdentityId`. It's a special format just for gestalt IDs and only used on the CLI.

Reply:
Actually scratch that, rename this to `parseProviderIdentityId`. I just saw the output.

But please move it to the appropriate location.

Reply:
Actually this was just incorrectly produced. The return type is just meant to be `['identity', ProviderIdentityId]`.

Reply:
This is now done.